CITATIONS.md

```diff
@@ -0,0 +1,38 @@
+# nf-core/proteomicslfq: Citations
+
+## Pipeline tools
```

Comment:
Maybe the nf-core framework too ;-)

Reply:
I think this is on the root README.md. Do you mean this?